package.json

```diff
@@ -12,8 +12,10 @@
     "dependencies": {
         "chalk": "4.1.0",
         "commander": "6.1.0",
+        "cson-parser": "^4.0.5",
```

Comment:
I get an error `npm ERR! 404 Not Found - GET http://my-registry-url/coffeescript/-/coffeescript-1.12.8.tgz` during installation. It comes from `cson-parser`. There are some mentions of this problem, for example at jashkenas/coffeescript/issues/4805, but it's not clear how to fix it while installing Angular Eslint with `ng add @angular-eslint/schematics`

Reply:
Hello, can you open a new issue in order to track the error with Angular ESLint + this package? Thank you :)

Reply:
- https://github.com/angular-eslint/angular-eslint/issues/194
- https://github.com/typescript-eslint/tslint-to-eslint-config/issues/839

Reply:
Thank you, we will look into these and provide a proper fix soon.